src/sentry/api/endpoints/organization_events.py

```diff
@@ -249,7 +249,7 @@ def get(self, request: Request, organization) -> Response:
             )
 
         on_demand_metrics_enabled = (
-            batch_features.get("organizations:on-demand-metrics-extraction", False)
+            batch_features.get("organizations:on-demand-metrics-extraction-widgets", False)
```

Comment:
Do we need this first check if we have control of the UI using `useOnDemand=true`?

Reply:
Okay.
WIthout changing that, does the PR look good as-is?

Reply:
`events` is fine to be widgets only, in the future if alerts somehow uses a table perhaps they add the alerts flag back here